1_fetch.yml

```diff
@@ -0,0 +1,76 @@
+target_default: 1_fetch
+
+packages:
+  - sbtools
+  - readr
+  - jsonlite
+
+sources:
+  - 1_fetch/src/fetch_sb_item_files.R
+
+targets:
+  1_fetch:
+    depends:
+      - 1_fetch/out/config.json
+      - 1_fetch/out/toha_irradiance_sb_files.rds
+      - 1_fetch/log/toha_irradiance_downloaded.rds
+      - 1_fetch/out/toha_temp_pred_sb_files.rds
+      - 1_fetch/log/toha_temp_pred_downloaded.rds
+      - 1_fetch/out/toha_clarity_sb_files.rds
+      - 1_fetch/log/toha_clarity_downloaded.rds
+  
+  toha_sbid:
+    command: c(I("5e5d0bb9e4b01d50924f2b36"))
+  clarity_sbid:
+    command: c(I("5e5d0b96e4b01d50924f2b34"))
+  
+  1_fetch/out/lake_metadata.csv:
+    command: download_sb_single_file(
+      target_name = target_name,
+      sb_id = I("5e5c1c1ce4b01d50924f27e7"), 
+      sb_filename = I('lake_metadata.csv'))
+  
+  1_fetch/out/config.json:
+    command: download_sb_single_file(
+      target_name = target_name,
+      sb_id = I("5e5c1c36e4b01d50924f27ea"), 
+      sb_filename = I('pb0_config.json'))
+  
+  1_fetch/out/toha_irradiance_sb_files.rds:
```

Comment:
I'm wondering if it makes sense just to skip the `1_fetch/out/toha_irradiance_sb_files.rds` targets. 
Reasons: 
 - This is a quick build target that you probably want to be able to manually trigger (e.g., dummy date). 
 - This target doesn't tell you if the sciencebase files have changed, so it isn't a good target to use to trigger a rebuild of the `_downloaded.rds` target
 - We don't use these targets for any other stages downstream.

So wondering why you don't want to list and download (and hash) the files all in one fetch step?